.travis.yml

```diff
@@ -18,3 +20,5 @@ services:
   - docker
 before_install:
   - (test $RUN_ON_LIVE_SERVER = 1 && ./.travis/start_hawkular_services.sh -d && ./.travis/wait_for_services.rb) || echo "Skipping live server"
+after_success:
+  - ./.travis.docs.rb
```

Comment:
I saw that doc but I would rather do the releases with Edgar deliberately, more than a side effect of a new tag. What if there is a tag that is not a release? @abonas what do you think?